index.php

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+     <head> 	
+     <title> Login Page </title> 
+     </head>
+     <body>
+	
+	<center><h1>Login Page</h1></center>
+	<div align="center" border>
```

Comment:
```suggestion
	<div align="center">
```
uninitialized or invalid property assigned as `border`